packages/backend/server/src/core/user/resolver.ts

```diff
@@ -70,6 +71,20 @@ export class UserResolver {
     };
   }
 
+  @Throttle('strict')
+  @Query(() => [PublicUserType], {
+    name: 'usersByIds',
+    description: 'Find users by ids',
+  })
+  @Public()
+  async findUsersByIds(
+    @Args('ids', { type: () => [String] }) ids: string[]
```

Comment:
look weird. maybe called by unlimited ids, to bypass throttler. we could provide a simple by id query. I've checked github apis a lot today, it also only provide user fetching by login, no batching options